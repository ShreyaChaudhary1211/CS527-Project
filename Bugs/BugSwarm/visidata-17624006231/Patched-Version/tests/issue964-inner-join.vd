sheet	col	row	longname	input	keystrokes	comment
			open-file	tests/issue964b.csv	o	
			open-file	tests/issue964a.csv	o	
issue964b	cod_istat		key-col		!	toggle current column as a key column
issue964a	domicilioCom		key-col		!	toggle current column as a key column
issue964a			join-sheets-top2	inner	&	concatenate top two sheets in Sheets Stack
