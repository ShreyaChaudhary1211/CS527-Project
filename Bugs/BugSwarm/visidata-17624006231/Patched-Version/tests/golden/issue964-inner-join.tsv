domicilioCom	sesso	domicilioProv	comune
082053	M	PA	Palermo
082053	M	PA	Palermo
082070	M	PA	Termini Imerese
082014	M	PA	Caccamo
